vulnerabilities/models.py

```diff
@@ -642,6 +644,152 @@ def get_absolute_url(self):
         """
         return reverse("package_details", args=[self.purl])
 
+    # TODO: There are other methods, variables etc. in models.py that need similar renaming.
+    def get_fixed_by_package_versions(self, fix=True):
```

Comment:
Since it deals with database object filtering, a better location for this logic would be on the `PackageQuerySet` manager.

Reply:
Here as well I don't follow your suggestion, which like the one just above (re the `affected()` queryset method) involves the `PackageQuerySet` class (manager?).  I haven't worked with those before and am not sure how to approach it.

Reply:
Thank you for all of your comments and suggestions @tdruez .  I still don't follow your last two comments, both of which involve queryset filters and the `PackageQuerySet()` class (which I gather is also referred to as the "manager").  

I'm going to see if I can find some material I can read and perhaps experiment with to get some understanding of how a queryset method in general, and our `PackageQuerySet()` manager in particular, are used.

Reply:
Code that deals with an object instance, here a Package, should exist as methods of the `Model`.

Code that deals specifically with database queries, such as returning multiple objects from filtering, should exist as methods of the `QuerySet/Manager` class.

See https://docs.djangoproject.com/en/4.2/topics/db/managers/ to get familiar with Manager as it's a key part of Django.